packages/trader/src/index.tsx

```diff
@@ -3,7 +3,16 @@ import { makeLazyLoader, moduleLoader } from '@deriv/shared';
 import { Loading } from '@deriv/components';
 
 const App = makeLazyLoader(
-    () => moduleLoader(() => import(/* webpackChunkName: "trader-app", webpackPreload: true */ './App/index')),
+    () =>
+        moduleLoader(() => {
+            const is_dtrader_v2 = JSON.parse(localStorage.getItem('FeatureFlagsStore') ?? '').data.dtrader_v2;
```

Comment:
if the `FeatureFlagsStore` is not present in localstorage, JSON.parse('') will throw an error. Please use try catch to parse JSON